dask_sql/physical/rex/core/call.py

```diff
@@ -867,6 +880,7 @@ class RexCallPlugin(BaseRexPlugin):
         "not like": NotOperation().of(LikeOperation()),
         "like": LikeOperation(),
         "similar to": SimilarOperation(),
+        "negative": InverseOperation(),
```

Comment:
For clarity, should we rename this to `NegativeOperation`? Inverse is a somewhat vague term which could indicate a different type of inverse from the _additive_ inverse we're computing here.

Reply:
yep, that is a good point